eng/pipelines/templates/steps/vcpkg.yml

```diff
@@ -6,8 +6,28 @@ parameters:
   # invocations
   DependenciesVariableName: vcpkg.deps
 
+  # Ref at which to check out (can be commit SHA or tag in the form of
+  # `tags/<tag>`)
+  VcpkgVersion: c6d69984edceacbe880fad20ddc8305a7ec6fb8c
```

Comment:
Why did we choose this commit hash? I think we should consider using a tagged release instead rather than some commit from master:
https://github.com/microsoft/vcpkg/releases

That means every 1-2 months, when a new release comes out, we know to update, and also know to which commit.

Reply:
This SHA was selected arbitrarily at the time that this change was made. Using tags is reasonable.